embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/crumbs/FragmentBreadcrumbDataSource.kt

```diff
@@ -0,0 +1,90 @@
+package io.embrace.android.embracesdk.capture.crumbs
+
+import io.embrace.android.embracesdk.arch.DataCaptureService
+import io.embrace.android.embracesdk.config.ConfigService
+import io.embrace.android.embracesdk.internal.clock.Clock
+import io.embrace.android.embracesdk.logging.InternalEmbraceLogger
+import io.embrace.android.embracesdk.logging.InternalStaticEmbraceLogger
+import io.embrace.android.embracesdk.payload.FragmentBreadcrumb
+import io.embrace.android.embracesdk.utils.filter
+import java.util.Collections
+
+/**
+ * Captures fragment breadcrumbs.
+ */
+internal class FragmentBreadcrumbDataSource(
```

Comment:
is this class tested?

Reply:
Yes, this is covered via the existing `EmbraceBreadcrumbServiceTest`